src/vs/code/electron-sandbox/issue/issueReporterPage.ts

```diff
@@ -85,6 +86,17 @@ export default (): string => `
 	</div>
 
 	<div class="system-info" id="block-container">
+		<div class="block block-extension-data">
+			<input class="send-extension-data" aria-label="${sendExtensionData}" type="checkbox" id="includeExtensionData" checked/>
+			<label class="extension-caption" id="extension-caption" for="includeExtensionData">
+				${sendExtensionData}
+				(<span id="ext-loading" hidden></span><a href="#" class="showInfo" id="extension-id">${escape(localize('show', "show"))}</a>)
```

Comment:
It's a tad bit wonky, but since this code here specifies a specific scheme for the `showInfo` type, I didn't want to repeat code just for this case. It prevents us from doing this:

```
<span ....>
<a href>...
<span...>,
```

https://github.com/microsoft/vscode/blob/49b6fa9a20c1fbbfb4db343c63b68d0941d3d69b/src/vs/code/electron-sandbox/issue/issueReporterService.ts#L259